templates/web/base/report/update.html

```diff
@@ -43,10 +43,14 @@ <h4 class="static-with-rule">[% loc('Updates') %]</h4>
                     </div>
                     [% END %]
 
-                  [% SET update_state = update.problem_state_display(c) %]
-                  [% IF update_state AND update_state != global.last_state AND NOT (global.last_state == "" AND update.problem_state == 'confirmed') %]
-                    <p class="meta-2">[% loc('State changed to:') %] [% update_state %]</p>
+                  [% SET update_state = update.problem_state %]
+                  [% IF ( update_state AND update_state != global.last_state AND NOT (global.last_state == "" AND update.problem_state == 'confirmed') ) OR
+                     update.mark_fixed OR update.mark_open
+                  %]
+                    <p class="meta-2">[% loc('State changed to:') %] [% update.problem_state_display(c) %]</p>
                     [%- global.last_state = update_state %]
+                    [%- IF update_state == "" AND update.mark_fixed %][% global.last_state = 'user - fixed' %][% END %]
```

Comment:
This should be `fixed - user`

Reply:
Just to note in case this ever bites us - it's theoretically possible for one update to have `fixed - user`, the next to have `fixed - council`, and then it would say a state change. There are a very small number of such cases, looks to be due to rare condition such as user and council leaving updates at the same time, so I don't think this is worth doing anything about, but perhaps that is why it previously used the `problem_state_display` for comparison and not `problem_state`.